### DISCUSSION TITLE
Do I need to sacrifice my .emacs.d to Radian?

### DISCUSSION THREAD

**Discussion by KingCode** [radian-software/radian#510] [Q&A]
This looks veryinteresting, and I am interested in exploring it, but I am afraid to muck with my `.emacs.d` (see below). Does Radian need to be installed in `.emacs.d`? It seems that is the case. 

I am currently using [Chemacs](https://github.com/plexus/chemacs2) to try out environments safely, which works well for me in trying new environments in parallel - without wrecking my working emacs config. So I would like to install radian in its own dir. and make that dir. `user-emacs-directory` for Chemacs to point to it, but from I have seen so far it looks like `user-emacs-directory` is the local user config within the Radian config.

Is there a way to install Radian and try it out without impacting `.emacs.d`? If not, I would like to suggest a future version with that capability, perhaps creating a new var `radian-local-user-emacs-directory` or something similar.

Thank you! 

PS> I added this as an issue as well - sorry for the duplicate.

**Comment by raxod502** [comment]
I'm sorry, for whatever reason I did not see a notification for this post until today(?!).

I agree completely that this should be a supported use case. Currently, I'm not sure whether or not it's possible. I've had the same struggle testing out Doom Emacs. I think any required changes to Radian would be pretty minimal, and I'd be happy to review them. Otherwise, this is one of many issues I will address when I have time to focus primarily on Radian.
